python/.pylint/accepted.rc

```diff
@@ -0,0 +1,556 @@
+[MASTER]
```

Comment:
QQ: how'd you pick the settings in this file (& in `.pylint/suggested.rc`)? is there a default PEP8 pylint file that we can/should diff this one against?

Reply:
I generated the default rc file that pylint uses and made incremental changes to `suggested.rc` and `accepted.rc`. I am adding a diff too, so that larger audience can see that. Thanks for pointing out!
```
(dev3) yo@C02VM7GBHTD5:~/code/spark-deep-learning$ pylint --generate-rcfile > python/.pylint/default.rc
No config file found, using default configuration
```